### DISCUSSION TITLE
Cockpit Session timeout

### DISCUSSION THREAD

**Discussion by yasirvohra** [cockpit-project/cockpit#16610] [Q&A] [answered]
Cockpit version: cockpit-bridge/focal, now 215-1 amd64 
OS: UBUNTU 20.04

Issue: we have installed cockpit and its great for system administration. I closed browser and left session open, just wondering it keeps the process running on background. 
I need help how to set cockpit session timeout automatically if I close browser directly or forget to logout session?

**Comment by martinpitt** [comment, marked as answer]
If you close the browser tab, the session should time out automatically after some 20 seconds. Just check `who` while you have an open cockpit session, looks something like this:

     martin   web console  2021-11-16 07:09 (::ffff:127.0.0.1)

Then close the browser tab, and run `watch who`, and the "web console" session will time out.
